pkg/kubernetes/job_test.go

```diff
@@ -84,5 +85,55 @@ var _ = Describe("Job", func() {
 	})
 
 	Describe("#Status", func() {
+		var s manifest.Status
+
+		BeforeEach(func() {
+			o := job.Object()
+			completions := int32(1)
+			o.Status.Succeeded = 1
+			o.Spec.Completions = &completions
```

Comment:
move completions := int32(1) to the top so you create the object and fill in the values after